cmd/cluster/core/create.go

```diff
@@ -330,6 +333,11 @@ func prototypeResources(opts *CreateOptions) (*resources, error) {
 		prototype.Cluster.Spec.Etcd.Managed.Storage.PersistentVolume.StorageClassName = ptr.To(opts.EtcdStorageClass)
 	}
 
+	if opts.EtcdStorageSize != "" {
+		etcdStorageSize := resource.MustParse(opts.EtcdStorageSize)
```

Comment:
MustParse will panic if the value is invalid, can we please return an error here instead.